plasmapy/diagnostics/langmuir.py

```diff
@@ -273,7 +273,8 @@ def swept_probe_analysis(probe_characteristic, probe_area, gas,
         Estimate of the ion saturation current in units of Am^-2.
 
     "hot_fraction" : float
-        Estimate of the total hot (energetic) electron fraction.
+        Estimate of the total hot (energetic) electron fraction. Only supplied
+        if bimaxwellian is True.
```

Comment:
This is minor, but I would call it a "bi-Maxwellian"

Reply:
That's a better name, but it won't work as an argument.